### DISCUSSION TITLE
UVtools Acceleration Guide Recommendation

### DISCUSSION THREAD

**Discussion by xPew** [sn4k3/UVtools#395] [Q&A] [answered]
I tried to use UVtools Acceleration Guide Recommendation, but I failed to do so.
I downloaded the code, made a file.g/file.gcode, and tried to run it on my Creality Halot One, but no file appeared in the menu.
Am I doing something wrong or it's related to the new proprietary creality firmware?

**Comment by sn4k3** [comment, marked as answer]
That mod are only known to work on Chitu-based machines. Halot is not one of them.
Nothing on that article will work on Creality machines

**Reply by xPew** [reply]
I see, thanks! And the other smart features like dynamic lift/dinamic exposure etc works?
